### DISCUSSION TITLE
Showing Week Numbers in Calendar Component

### DISCUSSION THREAD

**Discussion by apollocoder** [mantinedev/mantine#6200] [Feature requests]
Identifying the week number next to each row would be very helpful. 

![image](https://github.com/mantinedev/mantine/assets/2370188/7c7b2f2c-66ef-4249-9303-34f3f9656116)

**Comment by juliuslissau** [comment]
Yes!

**Comment by MariaBanaszkiewicz** [comment]
I will implement this feature! :)

**Reply by MariaBanaszkiewicz** [reply]
https://github.com/mantinedev/mantine/pull/7179 :)
